config/config.md

```diff
@@ -176,7 +176,9 @@ must be the same with the number of given `--state-sync-ids`.
 
 #### `--partial-sync-primary-network` (string)
 
-Partial sync enables non-validators to optionally sync only the P-chain on the primary network.
+Partial sync enables nodes that are not primary validators to optionally sync only
```

Comment:
nit: I think we typically say "primary network validators" rather than "primary validators"

past that lgtm